metacatalog/util/results.py

```diff
@@ -383,8 +383,8 @@ def get_data(self, verbose=False, merge=False, **kwargs) -> dict:
                 elif len(unmerged) > 0:
                     data[member.checksum] = unmerged
         
-        # Composites always try to merge
-        if self.group is not None and self.group.type.name == 'Composite':
+        # Composites and Split datasets always try to merge
+        if self.group is not None and self.group.type.name in ('Composite', 'Split dataset'):
```

Comment:
I think we need to handle Composites and Split datasets separately, as composites should be merged and split datasets should be concatenated